Lil-Peep/src/sections/featured-kits.liquid

```diff
@@ -0,0 +1,160 @@
+<section class="featured-kits">
+  <div class="flex-grid">
+    {% for block in section.blocks %}
+      {% if block.settings.opc == "1" %}
```

Comment:
Maybe you should use 'case', it is better to use when you have nested ifs.
Example:
```
{% for block in section.blocks %} 
    {% case block.settings.opc %} 
        {% when '1' %}
            do something...
        {% when '2' %}
            do something...
        {% when '3' %}
            do something...
        {% when '4' %}
            do something...
    {% endcase %} 
{% endfor %}
```